hedera-node/configuration/previewnet/application.properties

```diff
@@ -7,4 +7,4 @@
 bootstrap.genesisPublicKey=c249a323c878f5b5e2daccda6d731e6fdc32f870228d1cd4fae559d947dbc36c
 contracts.chainId=297
 ledger.id=0x02
-entities.unlimitedAutoAssociationsEnabled=true
+entities.unlimitedAutoAssociationsEnabled=true
```

Comment:
This change needs to be reverted because it may cause issues. All property-files must end with an empty line.

Reply:
Thank you! Great catch! Added the empty line.